scss/main.scss

```diff
@@ -1,1027 +1,1058 @@
-@import "./custom/_variables";
-@import "./bootstrap/bootstrap";
-@import "./instantsearch/instantsearch";
-@import "./instantsearch/instantsearch-theme-algolia";
-@import "./custom/includes/buttons";
-@import "./custom/includes/card";
-@import "./custom/includes/copy_to_clipboard";
-@import "./custom/includes/navbar";
-@import "./custom/includes/images";
-@import "./custom/includes/jumbotron";
-@import "./custom/includes/pagination";
-@import "./custom/includes/social_links";
-@import "./custom/includes/syntax_highlighting";
-@import "./custom/includes/tables";
-@import "./custom/includes/type";
+@import './custom/_variables';
+@import './bootstrap/bootstrap';
+@import './instantsearch/instantsearch';
+@import './instantsearch/instantsearch-theme-algolia';
+@import './custom/includes/buttons';
+@import './custom/includes/card';
+@import './custom/includes/copy_to_clipboard';
+@import './custom/includes/navbar';
+@import './custom/includes/images';
+@import './custom/includes/jumbotron';
+@import './custom/includes/pagination';
+@import './custom/includes/social_links';
+@import './custom/includes/syntax_highlighting';
+@import './custom/includes/tables';
+@import './custom/includes/type';
```

Comment:
You seem to have some sort of formatter or linter turned on that turned all of the `"` to `'` and has changed the style of indentations. 
I agree that linters/auto-formatters are great, but using it here has changed the entire file. 
Is this a commit that you are able to revert easily? If not, I'm happy to help with that.